packages/testing/src/PHPUnit/AbstractRectorTestCase.php

```diff
@@ -203,23 +203,26 @@ protected function doTestExtraFile(string $expectedExtraFileName, string $expect
         }
 
         $addedFilesWithNodes = $this->removedAndAddedFilesCollector->getAddedFilesWithNodes();
-        foreach ($addedFilesWithNodes as $addedFileWithNodes) {
-            if (! Strings::endsWith($addedFileWithNodes->getFilePath(), $expectedExtraFileName)) {
+        foreach ($addedFilesWithNodes as $addedFilesWithNode) {
+            if (! Strings::endsWith($addedFilesWithNode->getFilePath(), $expectedExtraFileName)) {
                 continue;
             }
 
-            $printedFileContent = $this->betterStandardPrinter->prettyPrintFile($addedFileWithNodes->getNodes());
+            $printedFileContent = $this->betterStandardPrinter->prettyPrintFile($addedFilesWithNode->getNodes());
             $this->assertStringEqualsFile($expectedExtraContentFilePath, $printedFileContent);
             return;
         }
 
         $movedFilesWithContent = $this->removedAndAddedFilesCollector->getMovedFileWithContent();
-        foreach ($movedFilesWithContent as $movedFileWithContent) {
-            if (! Strings::endsWith($movedFileWithContent->getNewPathname(), $expectedExtraFileName)) {
+        foreach ($movedFilesWithContent as $singleMovedFilesWithContent) {
```

Comment:
This should be handled differently, as files goes to file.

Reply:
is Inflector service need extract camel case with end "s" or "es" on first part to check it?

Reply:
Hmm... maybe. Or we could rename `MovedFileWithContent` to comply with this naming standard.
Any ideas?

Reply:
It seems we need to extends the `Doctrine\Inflector\Inflector` service as I realized it came from vendor.

Reply:
implemented.